apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net.csproj

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I considered that, but figured it would be easier to customize things if everything was self-contained - as we're generating the projects, it's easy enough to modify the generator.

On the other hand, using a common properties file would help in a *few* cases. Hmm. Will ponder that - it's easy enough to move to that later.